documentation/firmware-management/index.rst

```diff
@@ -78,7 +77,7 @@ Hyperview uses the Firmware Version property (asset → *Information → Propert
 
 In both contexts, an asset is considered to have "managed firmware" only if the asset's firmware group is mapped in Hyperview Product Catalog. (A "firmware group" consists of firmware versions for a specific product model.)
```

Comment:
I suggest keep the term same. Either "Hyperview Catalog" or "Hyperview Product Catalog".

Reply:
Agreed. I made the change.